packages/interfaces/src/service/navigation-service.interface.ts

```diff
@@ -64,9 +64,8 @@ export interface Navigator {
      * If the blocker function returns true navigation is blocked. If it returns
      * false navigation will proceed as normal.
      *
-     * A location is provided to the blocker function. This could be used to later
-     * on resume the navigation, for instance as the result of some user response or
-     * equivalent.
+     * To later on resume navigation that was blocked as the result of some user
+     * response, you may save away the new location provided to the blocker function.
```

Comment:
I think what is meant here is "to resume, as the result of some user response" rather than "blocked as the result of some user response". Maybe just remove that?

```suggestion
     * To later on resume navigation that was blocked,
     * you may save away the new location provided to the blocker function.
```